collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -306,6 +314,14 @@ public synchronized Tracker getDefaultTracker() {
         return tracker;
     }
 
+    private boolean firstAppRun() {
```

Comment:
This could be pretty misleading. It's not actually the first app run, it's the first app run after upgrading to the version in which the flag was introduced, right? Seems the naming could cause problems in the future but I don't have a great alternate proposal.

Reply:
Yes a I think I should add a short comment here. It's first app run or first app run after upgrading to the version in which the flag was introduced as you said.